src/compiler/typescript_to_rust/azle_generate/src/generators/header/mod.rs

```diff
@@ -1,4 +1,8 @@
+mod timers;
+
 pub fn generate_header_code() -> proc_macro2::TokenStream {
+    let timers_module = timers::generate_timers_module();
```

Comment:
It's strange to me that the timers code is in the header. It seems architecturally like the timers code is just like any other code and has no reason to be in the header. It will be confusing to have to go to the header section to modify the timers code. I understand maybe you had to do this because of the module's scope? I would like to address this